imports/ofx/account.go

```diff
@@ -0,0 +1,58 @@
+package ofx
+
+import (
+	"github.com/aclindsa/xml"
+	"gorm.io/gorm"
+)
+
+type Account struct {
+	gorm.Model
+	Currency     string
+	BankID       string
+	AccountID    string
+	Type         string
+	Transactions []Transaction
+}
+
+func (Account) TableName() string {
+	return tablePrefix + "accounts"
+}
+
+func (a *Account) UnmarshalXML(decoder *xml.Decoder, start xml.StartElement) error {
+	type Alias Account
+
+	var data struct {
+		Alias
+		AccountInfo struct {
+			Currency string `xml:"CURDEF"`
+			Meta     struct {
+				BankID    string `xml:"BANKID"`
+				AccountID string `xml:"ACCTID"`
+				Type      string `xml:"ACCTTYPE"`
+			} `xml:"BANKACCTFROM"`
+			TransactionsStruct struct {
+				Transactions []Transaction `xml:"STMTTRN"`
+			} `xml:"BANKTRANLIST"`
+		} `xml:"STMTRS"`
+	}
+
+	if err := decoder.DecodeElement(&data, &start); err != nil {
+		return err
+	}
+
+	*a = Account(data.Alias)
```

Comment:
as I see, we don't parse any fields into the `Alias` struct we embed and use only those specified manually in the inlined struct. Do we need an `Alias` here?
```suggestion
	var data struct {
		AccountInfo struct {
			Currency string `xml:"CURDEF"`
			Meta     struct {
				BankID    string `xml:"BANKID"`
				AccountID string `xml:"ACCTID"`
				Type      string `xml:"ACCTTYPE"`
			} `xml:"BANKACCTFROM"`
			TransactionsStruct struct {
				Transactions []Transaction `xml:"STMTTRN"`
			} `xml:"BANKTRANLIST"`
		} `xml:"STMTRS"`
	}

	if err := decoder.DecodeElement(&data, &start); err != nil {
		return err
	}
```